packages/bpk-component-star-rating/src/BpkStar.module.scss

```diff
@@ -26,7 +26,9 @@
 }
 
 .bpk-star {
-  display: inline-block;
+  display: inline-flex;
```

Comment:
Not 100% sure about the use of inline-flex in a non flex styled element.  But it does sort the strange behaviour in firefox the alternative is to redo it all in flex.